smoke-tests/src/test/java/io/opentelemetry/smoketest/AbstractTestContainerManager.java

```diff
@@ -27,6 +27,8 @@ protected Map<String, String> getAgentEnvironment(
         jvmArgsEnvVarName,
         "-Xmx512m -javaagent:/"
             + TARGET_AGENT_FILENAME
+            // args passed to the agent directly
+            + "=otel.javaagent.debug=true;otel.bsp.schedule.delay=10ms"
```

Comment:
This doesn't really make sense to me, even if this doesn't work at all will something fail? Perhaps it would make sense to restore the original behavior and here use something like `=otel.instrumentation.http.server.capture-request-headers=X-Test-Request` that could be verified in AppServerTest?

Reply:
good idea